doc/specs/top_10.md

```diff
@@ -0,0 +1,92 @@
+# Top 10
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+Users are notified of the top 10 most popular items on the website. Users can view the list in a homepage widget or on a separate dedicated page. Users can see the most popular items by past day, past week, past month, and all-time.
+
+## Business rules
+
+- Users can enable or disable the top 10 list from displaying on their homepage.
+- Each item in the list must contain information relevant to the item.
+- The list will be automatically updated by a ranking algorithm.
+- The list update interval will be configured in the settings.json file.
+
+## Use cases
+
+### 1.0 - Modifying the Top 10 widget
```

Comment:
Does it make sense that the homepage section would only show the Top 10, I can't imagine people would want 100 or 250 items on their homepage.